pkgs/by-name/xa/xan/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "xan";
+  version = "0.42.0";
+
+  src = fetchFromGitHub {
+    owner = "medialab";
+    repo = "xan";
+    tag = version;
+    hash = "sha256-8uNfiPJjZWqtbrBU/LRYM7PI5/4JBqZGe1dtCWArU8M=";
+  };
+
+  cargoHash = "sha256-4OVDiXa8iutoPr816yjMFYa+bC5vZ+iHHrij7uwYUZ0=";
+  enableParallelBuilding = true;
```

Comment:
I'm curious, why is this option necessary? On my machine, the same number of cores is used with or without it and the build finishes at nearly the same times.

Reply:
Looks like I forgot to remove it from my own testing. Removed in a newer commit.